apis/placement/v1beta1/work_types.go

```diff
@@ -47,6 +49,9 @@ const (
 type WorkSpec struct {
 	// Workload represents the manifest workload to be deployed on spoke cluster
 	Workload WorkloadTemplate `json:"workload,omitempty"`
+
+	// WorkloadOverrides represents a list of overrides applied to the resources.
+	WorkloadOverrides []WorkloadOverride `json:"workloadOverrides,omitempty"`
```

Comment:
why do we need this? Which component is going to apply the overrides?

Reply:
Right now hubagent will apply the override and then create the manifest. memberAgent need the applyMode info to know how to apply the resource, eg, createOnly or server-side apply.

In addition, this field provides the troubleshooting information.